### PR TITLE
added CI pipeline for vm module

### PR DESCRIPTION
**Pull request by alexkonkin**

### CODE REVIEW

File: infra/concourse/pipelines/terraform-google-vm.yml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+          platform: linux
+          inputs:
+            - name: pull-request
+              path: terraform-google-vm
+          run:
+            path: make
+            args:
+              - '-s'
+              - test_integration
+            dir: terraform-google-jenkins
```
Comment (aaron-lane) [inline infra/concourse/pipelines/terraform-google-vm.yml]:
```suggestion
            dir: terraform-google-vm
```

File: infra/concourse/pipelines/terraform-google-vm.yml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+          status: pending
+      - get: lint-test-image
+        trigger: true
+      - task: run
+        image: lint-test-image
+        config:
+          platform: linux
+          inputs:
+            - name: pull-request
+              path: terraform-google-jenkins
```
Comment (aaron-lane) [inline infra/concourse/pipelines/terraform-google-vm.yml]:
```suggestion
              path: terraform-google-vm
```

Comment (alexkonkin) [reply infra/concourse/pipelines/terraform-google-vm.yml]: Aaron, thank you for your review. I've fixed my errors.

File: infra/terraform/test_fixtures/terraform-google-vm.tf [hunk 3]
```diff
@@ -0,0 +1,99 @@
+
+cals {
```
Comment (aaron-lane) [inline infra/terraform/test_fixtures/terraform-google-vm.tf]:
```suggestion
locals {
```

Comment (alexkonkin) [reply infra/terraform/test_fixtures/terraform-google-vm.tf]: Thank you and my apologies for this error.

**Pull request merged**